src/leo-server-app/src/models/command.ts

```diff
@@ -33,4 +33,4 @@ export const commandSchema = new Schema(
 
 const Command = mongoose.model("Command", commandSchema);
 
-module.exports = Command;
+export default Command;
```

Comment:
is there a particular reason to change these?

Reply:
For some reason, the export it like this and importing it using require makes the export lose its type